cmd/collector/app/http_handler.go

```diff
@@ -29,11 +28,17 @@ import (
 )
 
 const (
-	formatParam = "format"
 	// UnableToReadBodyErrFormat is an error message for invalid requests
 	UnableToReadBodyErrFormat = "Unable to process request body: %v"
 )
 
+var (
```

Comment:
For my own improvement: why the parenthesis?

Reply:
No strong reason, it looks consistent with const block as declarative section.